gear_ratio: 48
wheel_radius: 0.03
wheel_sep: 0.135
